src/monorepo-workflow-operations.test.ts

```diff
@@ -1,1788 +1,1214 @@
-                    TODAY: '2022-06-12',
-                  },
-                  validateReleaseSpecification: {
-                    packages: {
-                      a: releaseSpecificationModule.IncrementableVersionParts
-                        .major,
-                      b: releaseSpecificationModule.IncrementableVersionParts
-                        .minor,
-                      c: releaseSpecificationModule.IncrementableVersionParts
-                        .patch,
-                      d: new SemVer('1.2.3'),
-                    },
-                  },
-                });
-
-                await followMonorepoWorkflow({
-                  project,
-                  tempDirectoryPath: sandbox.directoryPath,
-                  firstRemovingExistingReleaseSpecification: false,
-                  stdout,
-                  stderr,
-                });
-
-                expect(
-                  generateReleaseSpecificationTemplateForMonorepoSpy,
-                ).toHaveBeenCalled();
-                expect(updatePackageSpy).toHaveBeenNthCalledWith(1, {
-                  project,
-                  packageReleasePlan: {
-                    package: project.rootPackage,
-                    newVersion: '2022.6.12',
-                    shouldUpdateChangelog: false,
-                  },
-                  stderr,
-                });
-                expect(updatePackageSpy).toHaveBeenNthCalledWith(2, {
-                  project,
-                  packageReleasePlan: {
-                    package: project.workspacePackages.a,
-                    newVersion: '2.0.0',
-                    shouldUpdateChangelog: true,
-                  },
-                  stderr,
-                });
-                expect(updatePackageSpy).toHaveBeenNthCalledWith(3, {
-                  project,
-                  packageReleasePlan: {
-                    package: project.workspacePackages.b,
-                    newVersion: '1.1.0',
-                    shouldUpdateChangelog: true,
-                  },
-                  stderr,
-                });
-                expect(updatePackageSpy).toHaveBeenNthCalledWith(4, {
-                  project,
-                  packageReleasePlan: {
-                    package: project.workspacePackages.c,
-                    newVersion: '1.0.1',
-                    shouldUpdateChangelog: true,
-                  },
-                  stderr,
-                });
-                expect(updatePackageSpy).toHaveBeenNthCalledWith(5, {
-                  project,
-                  packageReleasePlan: {
-                    package: project.workspacePackages.d,
-                    newVersion: '1.2.3',
-                    shouldUpdateChangelog: true,
-                  },
-                  stderr,
-                });
-              });
-            });
+    describe('when firstRemovingExistingReleaseSpecification is false and the release spec file already exists', () => {
```

Comment:
Thoughts on whether we should have a test to ensure it doesn't open the editor?

Reply:
Ah, makes sense. Added in [fbc5bf4](https://github.com/MetaMask/create-release-branch/pull/21/commits/fbc5bf415790580d82b360e9971d38a78b63c448).